packages/babel-plugin-remove-graphql-queries/src/__tests__/index.js

```diff
@@ -9,6 +9,53 @@ function matchesSnapshot(query) {
   expect(code).toMatchSnapshot()
 }
 
+it(`Transforms queries in useStaticQuery`, () => {
+  matchesSnapshot(`
```

Comment:
Not related to this PR, but want to jot it down so we don't forget...

I don't actually think these tests are _all that_ valuable. The tests should validate something more meaningful, e.g. we should get the result of a transform, and then validate that e.g. an import statement appears and that import statement matches the hash of the data that's replaced.

It's harder to trace changes with these snapshots, and I don't think they're all that helpful.